theatre/studio/src/panels/DetailPanel/ProjectDetails.tsx

```diff
@@ -55,7 +55,7 @@ const ProjectDetails: React.FC<{
     }, 40000)
   }, [])
 
-  const [tooltip, openExportTooltip] = usePopover(
+  const {node: tooltip, open: openExportTooltip} = usePopover(
```

Comment:
Suggestion: Rename these to not unstructured `const exportTooltip = usePopover(` and in call sites just use `exportTooltip.open(node, mouseEvent)` or whatever.

Reply:
Oh I agree, I just didn't want the changelog to be a mess by also modifying call-sites of open/close etc

Reply:
Done